test/suite/test_tiered07.py

```diff
@@ -130,6 +131,25 @@ def test_tiered(self):
         self.session.drop(self.localuri)
         self.session.drop(self.uri)
 
+        # By default, the remove_files configuration for drop is true. This means that the
+        # drop operation for tiered tables should both remove the files from the metadata
+        # file and remove the corresponding local object files in the directory. Currently the
+        # below code is commented as the files are not removed from the directory. FIXME: WT-9003
+        # self.assertFalse(os.path.isfile("abc-0000000001.wtobj"))
+        # self.assertFalse(os.path.isfile("abc-0000000002.wtobj"))
```

Comment:
We should look into this as the drop call should remove the local objects. See my suggestion in WT-9003.